src/remember.ts

```diff
@@ -0,0 +1,71 @@
+import { Operator, StrictSink } from './types';
+
+export function remember<T>(
+  numRemembered: number = 1,
+  shouldUnsubscribe: boolean = false
```

Comment:
This should work! I think functions with boolean arguments are a bit inferior to named alternatives like `remember` (never unsubscribes) & `rememberReusable` (unsubscribes). Or `rememberPersistent` (never unsubscribes) & `remember` (unsubscribes). Something like that.

But it also seems like this is something that can be done later. I think the default you chose now is good and with usage we can learn what people expect. It might not even be the case that they will need to pass `true` as the second arg.

So, let's ship it.